### ISSUE TITLE
Rename text within Settings tab of the Affiliate Area

### ISSUE THREAD

**Issue by sumobi** [AffiliateWP/AffiliateWP#2176]
The `Settings` tab text doesn't match what's contained inside it. If someone clicks on the settings tab they're expecting to see "general" settings, however when you click on it you see "Profile Settings". One of the settings, enabling new referral notifications, doesn't sound like a profile option. To me, profile settings are those that are discussed here: https://github.com/AffiliateWP/AffiliateWP/issues/1513. First name, last name, email etc.

![screen shot 2017-04-20 at 11 18 24 am](https://cloud.githubusercontent.com/assets/52581/25206281/184e91ac-25bb-11e7-9512-384bc47cd15d.png)

I'd like to see:

1. The tab heading text change from `Profile Settings` to `Settings` 
2. The button text change from `save profile settings` to `save settings`
3. The save notice message changed from `Your affiliate profile has been updated` to `Your settings have been updated`

This would make the tab more like an actual "Settings" area. We could then allow for sub headings if need be. Example, for https://github.com/AffiliateWP/AffiliateWP/issues/1513 we might decide to create a "Update your profile" heading and then have all the fields underneath it, or we might decide to have all this under a "Profile" tab (payment email would move to this tab).

It also allows other add-ons to add their settings (if need be) to the tab and still make sense. Direct Link Tracking for example (it's being moved to its own tab) has its current settings there. When store credit supports allowing the affiliate to choose their payout option, it might make sense to add the setting to the settings tab, rather than creating an entire `Store Credit` tab. Maybe in the future an affiliate should choose to receive other notifications, these could also live under the settings tab.

Anyone agree/disagree?

**Comment by DrewAPicture** [comment]
I agree with points 1-3. As for add-ons adding their own settings there's utility in it, but I'll reiterate my objection (raised in [#2158](https://github.com/AffiliateWP/AffiliateWP/issues/2158#issuecomment-293934580)) intermingling add-on settings with AffWP core settings.

**Comment by sumobi** [comment]
I don't see any reason why a 3rd party add-on couldn't add their own setting (especially if there's just 1) to the existing settings tab. After all, we do provide a hook for it (affwp_affiliate_dashboard_before_submit) and it would be better to keep all settings, core or not under one tab, but utilize sub headings to distinguish them.

I do agree that 3rd party add-ons shouldn't use the same option row prefix as core however.

**Comment by pippinsplugins** [comment]
As long as the settings being added are appropriate, I don't have an issue with them being registered in the existing Settings tab.

I agree on 1-3.
